xml/geo_manage_hawk2_i.xml

```diff
@@ -0,0 +1,316 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect2
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+
+<sect2 
+ xml:id="sec.ha.geo.manage.hawk2"
+ xmlns="http://docbook.org/ns/docbook" 
+ xmlns:xi="http://www.w3.org/2001/XInclude" 
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" >
+ <title>With the &haweb; (&hawk;2)</title>
+
+ <info>
+      <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+        <dm:maintainer></dm:maintainer>
+        <dm:status>editing</dm:status>
+        <dm:deadline></dm:deadline>
+        <dm:priority></dm:priority>
+        <dm:translation></dm:translation>
+        <dm:languages></dm:languages>
+        <dm:release></dm:release>
+        <dm:repository></dm:repository>
+      </dm:docmanager>
+ </info>
+ &hawk-versions;
+ 
+  <sect3 xml:id="sec.ha.geo.manage.hawk2.dashboard">
+<!--taroth 2014-08-27: CAVE - the following is copied from ha_config_hawk.xml,
+ find a better solution next time-->
+   <title>Monitoring Multiple Clusters with the &hawk;2 Dashboard</title>
+   <para><remark>taroth 2015-10-29: krig: is the following still true?</remark>
+    The cluster information displayed in the <guimenu>Dashboard</guimenu> 
+    is stored in a persistent cookie. This means you
+    need to decide which &hawk;2 instance you want to view the
+    <guimenu>Dashboard</guimenu> on, and always use that one. The
+    machine you are running &hawk;2 on does not even need to be part of
+    any cluster for that purpose&mdash;it can be a separate, unrelated
+    system.
+   </para>
+  <procedure xml:id="pro.ha.config.hawk2.geo.dashboard">
+   <title>Monitoring Multiple Clusters with &hawk;2</title>
+   <para><remark>taroth 2015-10-29: krig: please check if all of them are still true!</remark></para>
+   <itemizedlist>
+    <title>Prerequisites</title>
+    <listitem>
+     <para> All clusters to be monitored from &hawk;'s <guimenu>Dashboard</guimenu> must be
+      running &productname; &productnumber;. It is not possible to monitor clusters that are
+      running earlier versions of &productname;. </para>
+    </listitem>
+    <listitem>
+     <para>If you did not
+      replace the self-signed certificate for &hawk;2 on every cluster node with your own
+      certificate (or a certificate signed by an official Certificate Authority), log in to
+      &hawk;2 on <emphasis>every</emphasis> node in <emphasis>every</emphasis> cluster at least
+      once. Verify the certificate (or add an exception in the browser to bypass the warning).
+     </para>
+    </listitem>
+    <listitem>
+     <para>If you are using Mozilla Firefox, you must change its preferences to <guimenu>Accept third-party
+       cookies</guimenu>. Otherwise cookies from monitored clusters will not be set, thus preventing
+      login to the clusters you are trying to monitor. </para>
```

Comment:
This, on the other hand, is no longer an issue since the dashboard no longer uses any cookie.